app/assets/javascripts/discourse/app/components/modal/second-factor-edit-security-key.js

```diff
@@ -0,0 +1,36 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+
+export default class SecondFactorEditSecurityKey extends Component {
+  @tracked loading = false;
+  onClose = this.args.model.onClose;
```

Comment:
@davidtaylorhq I am caching the `onClose` argument as we lose context of the args when in the `willDestroy` hook.